.fixtures.yml

```diff
@@ -0,0 +1,6 @@
+# This file can be used to install module dependencies for unit testing
+# See https://github.com/puppetlabs/puppetlabs_spec_helper#using-fixtures for details
+---
+fixtures:
+  forge_modules:
+#     stdlib: "puppetlabs/stdlib"
```

Comment:
If the stdlib dependency is commented out, it may be as well just taking it out.

Reply:
addressed